doc/Command-Reference.md

```diff
@@ -222,8 +222,6 @@
 * [Static DNS Commands](#static-dns-commands)
   * [Static DNS config command](#static-dns-config-command)
   * [Static DNS show command](#static-dns-show-command)
-* [Wake-on-LAN Commands](#wake-on-lan-commands)
```

Comment:
We should keep these doc.
Only revert the code.

Reply:
OK, will save the doc
https://github.com/sonic-net/sonic-utilities/pull/3516